docs/fundamentals/code-analysis/overview.md

```diff
@@ -54,36 +56,51 @@ The following rules are enabled, by default, in .NET 7.
 | [CA2259](quality-rules/ca2259.md) | Usage | `ThreadStatic` only affects static fields |
 | [CA2260](quality-rules/ca2260.md) | Usage | Use correct type parameter |
```

Comment:
These two are missing the Severity column.

Reply:
Good catch!